### PR TITLE
removing except value error

### PR DESCRIPTION
**Pull request by Benkdel**
removing ValueError

### CODE REVIEW

File: models/engine/file_storage.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def reload(self):
                 jo = json.load(f)
             for key in jo:
                 self.__objects[key] = classes[jo[key]["__class__"]](**jo[key])
-        except ValueError:
+        except:
```
Comment (CarlosGBentinDN) [inline models/engine/file_storage.py:58]: better to use finally: than to use except without any specific catch

Comment (Benkdel) [reply models/engine/file_storage.py:58]: thanks

Comment (Benkdel) [reply models/engine/file_storage.py]: implemented

**Review by CarlosGBentinDN — APPROVED** [review]
great!

**Pull request merged**